index.html

```diff
@@ -4297,19 +4304,60 @@ <h2>JSON Processing</h2>
 implementing the rule above. This can ensure proper term identification,
 typing, and order, when a JSON document is processed as JSON-LD.
           </p>
+          <p>
+However, the intention of the issuer is not preserved, unless the <code>@context</code>
+is applied, and the resulting <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>,
+and their exact RDF types, and node and edge structure is understood by the verifier.
+          </p>
 
           <p>
-The rule above guarantees semantic interoperability between JSON and JSON-LD for
-literal JSON keys mapped to URIs by the `@context` mechanism. While JSON-LD
-processors will use the specific mechanism provided and can verify that all
-terms are correctly specified, JSON-based processors implicitly accept the same
-semantics without performing any JSON-LD transformations, but instead by
+These rules alone do not guarantee semantic interoperability between JSON and JSON-LD for
+literal JSON keys mapped to URIs by the `@context` mechanism.
+          </p>
+While some RDF processors will use the language features of JSON-LD and can verify that all
+terms are correctly specified, JSON-LD processors explicitly accept the same
+semantics without performing any RDF transformations, but instead by
 applying the above rules. In other words, the context in which the data exchange
-happens is explicitly stated for both JSON and JSON-LD by using the same
-mechanism. With respect to JSON-based processors, this is achieved in a
-lightweight manner, without having to use JSON-LD processing libraries.
+happens is explicitly stated for both RDF and JSON-LD by using the same
+mechanism. With respect to JSON-LD-based processors, this is achieved in a
+lightweight manner, without having to use RDF processing libraries.
+          </p>
+        </section>
+        <section>
+          <h2>Advanced RDF Processing</h2>
```

Comment:
Can we find an alternative subtitle? There is nothing "Advanced" in what is in the section, at least no from an RDF point of view...

Reply:
The intention was to clearly communicate that there is "JSON-LD processing" where you don't apply the context (This was previously called "JSON Processing"... and there is JSON-LD processing, where you do apply the context (this is currently titled "Advanced RDF Processing".).

I'm not attached to the names, I am attached to signaling that both processing rules are in the context of RDF.

Reply:
I would not call the first "JSON-LD Processing", because it is very misleading. The term "JSON-LD Processing" is a term coined by the JSON-LD WG and [has a well-defined meaning](https://www.w3.org/TR/json-ld11-api/). It is actually closer to what you call "Advanced RDF Processing".

What we describe in the spec handling VC files in JSON-LD format but _without_ a full JSON-LD Processing is, though fairly general, proprietary to the VC specification. Maybe using something like "Basic VC Processing" would do.

Reply:
Manu suggested calling in RDF processing, I think thats a fair compromise... its the "JSON-LD... but with transformations that care about effort put into the context" processing kind...

Reply:
@dlongley suggested "Static Processing" vs. "Dynamic Processing"... where "Static Processing" covers things like "Use a JSON Schema to check the well-formed-ness of the payload and then process it like you would a JSON payload... and where "Dynamic Processing" is using a JSON-LD processor to compact/expand or convert to NQuads.

Thoughts?

Reply:
I am not sure about the "Dynamic" vs. "Static". I do not lie down on the road over this, but these two terms do not speak to me in this context.